source/slang/reflection.cpp

```diff
@@ -886,3 +886,22 @@ SLANG_API SlangReflectionEntryPoint* spReflection_getEntryPointByIndex(SlangRefl
 
     return convert(program->entryPoints[(int) index].Ptr());
 }
+
+SLANG_API SlangUInt spReflection_getGlobalConstantBufferBinding(SlangReflection* inProgram)
+{
+    auto program = convert(inProgram);
+    if (!program) return 0;
+    auto cb = program->globalScopeLayout->FindResourceInfo(LayoutResourceKind::ConstantBuffer);
```

Comment:
Again, these aren’t going to work for Vulkan, and I suspect they are subtly wrong for D3D as well. I’ll follow up on this topic later (I’m reviewing this on my phone).